### PR TITLE
46 duplicate copies of scripts to generate cicognara static site

### LINKED ISSUE #46: Duplicate copies of scripts to generate cicognara-static site
There are two copies of the scripts to generate the static site:
* `/lib`
* `/apps/cicognara-static/_bin`

We should pick one and remove the other.

Also there are two (AFAICT) unusued directories in `/apps`:
* `/apps/cicosearch`
* `/apps/site`

Can we remove those? `cicosearch` in particular trips me up with tab completion every time.

Items Ming identified with currently-deployed site (built using the `/apps/cicognara-static/_bin` scripts:
- [x] /catalogo/itempages/2962.html does not display Harvard's title
- [ ] /catalogo/itempages/3698.html same
- [ ] /catalogo/itempages/4.html no digital content (was working before)

### PR DESCRIPTION
**Pull request by escowles**
Fixes #46

### CODE REVIEW

File: apps/cicognara-static/_site/catalogo/itempages/1.html [hunk 1]
```diff
@@ -16,15 +16,10 @@
       <main id="main">
          <div class="container">
             <section class="catalogo-item-header">
-               <h1>1.
-                  
-                  	      AGINCOURT (d’) Seroux, Histoire de l’Art
```
Comment (escowles) [inline apps/cicognara-static/_site/catalogo/itempages/1.html:21]: This change is a little worrying — the entry being deleted is definitely correct, and the substituted text below appears much further down as a child of another entry — maybe an XPath issue?

File: apps/cicognara-static/_site/catalogo/itempages/10.html [hunk 2]
```diff
@@ -16,8 +16,7 @@
       <main id="main">
          <div class="container">
             <section class="catalogo-item-header">
-               <h1>10.
```
Comment (escowles) [inline apps/cicognara-static/_site/catalogo/itempages/10.html:19]: We should keep the item numbers.

**Comment by escowles** [comment]
@cwulfman I saw a branch but not a PR, so I opened this PR for review. I think it mostly looks good, but I do see some issues. I left comments about entry `#`1 and missing entry numbers. But I also see that a lot of the pages have digital content in `main` but don't here (e.g., `#`2 and `#`3)

File: apps/cicognara-static/_site/catalogo/itempages/1.html (hunk 1 above)
Comment (cwulfman) [reply apps/cicognara-static/_site/catalogo/itempages/1.html:21]: very worrying. I also don't know what the empty TEI files are doing there.  I'll review the generation scripts again

**Pull request merged**